Rubberduck.Parsing/VBA/COMReferenceManagerBase.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using NLog;
+using System.Collections.Concurrent;
+using Rubberduck.VBEditor;
+using System.Runtime.InteropServices;
+using System.IO;
+using Rubberduck.Parsing.ComReflection;
+
+namespace Rubberduck.Parsing.VBA
+{
+    public abstract class COMReferenceManagerBase : ICOMReferenceManager
+    {
+        private const string rubberduckGUID = "{E07C841C-14B4-4890-83E9-8C80B06DD59D}";
```

Comment:
That's the RD GUID defined in AssemblyInfo.cs; not sure it's a good idea to have it in two places. Mind you a `const` here is much better than the inline hard-coded string literal it was before... perhaps we should acquire that GUID through reflection?